src/emulator/io/include/io/state.h

```diff
@@ -46,14 +49,14 @@ enum TtyType {
 
 typedef std::map<SceUID, TtyType> TtyFiles;
 typedef std::map<SceUID, FilePtr> StdFiles;
-typedef std::map<SceUID, ZipFilePtr> ZipFiles;
+typedef std::map<SceUID, ReadOnlyInMemFile> AppFiles;
```

Comment:
Why no just AppFilePtr?

Reply:
I dont think the name AppFilePtr suits for it, since the class is just storing data using std::vector, it doesnt point to anything

Reply:
`AppFileBuffer`?

Reply:
better